invenio_cli/commands/requirements.py

```diff
@@ -113,7 +115,7 @@ def check_docker_version(cls, major, minor=-1, patch=-1, exact=False):
         # Output comes in the form of
         # 'Docker version 19.03.13, build 4484c46d9d\n'
         try:
-            result = run_cmd(["docker", "--version"])
+            result = run_cmd(["docker", "version"])
```

Comment:
update comment in line 19

Reply:
`docker version` outputs the version like this:

```
Client: Docker Engine - Community
 Version:           24.0.2
 API version:       1.43
 ...
```

as opposed to `docker -version`: `Docker version 24.0.2, build cb74dfc`

The classmethod `_version_from_string` works because it selects the first regex match, but I think it would actually be better to keep the `--version` query. 

Or which comment are you referring to?

Reply:
A more "reliable" alternative would be to use `docker version --format json` which returns JSON with the version string under the `Client.Version` key

Reply:
This would mean that we would have to chamnge [`check_docker_version`](https://github.com/inveniosoftware/invenio-cli/blob/master/invenio_cli/commands/requirements.py#L111) to something like this, should I add this to this PR?

```python
@classmethod
def check_docker_version(cls, major, minor=-1, patch=-1, exact=False):
    """Check the docker version."""
    # Use JOSN Formatted output and parse it
    try:
        result = run_cmd(["docker", "version", "--format", "json"])
        result_json = json.loads(result.output.strip())
        version = cls._version_from_string(result_json["Client"]["Version"])
        return cls._check_version("Docker", version, major, minor, patch, exact)
    except Exception as err:
        return ProcessResponse(error=f"Docker not found. Got {err}.", status_code=1)         
```

Reply:
LGTM and I think it brings us a bit closer to having a "stable" way of parsing the output, and if the option is there by Docker, I imagine they plan to keep it around.